src/main/java/com/startup/model/Student.java

```diff
@@ -0,0 +1,112 @@
+package com.startup.model;
+
+public class Student {
+
+    private long studentId;
+    private String firstName;
+    private String lastName;
+    private String gender;
+    private String cnp;
```

Comment:
If in class Teacher cnp is long, maybe you need use the same type of variable or it is need that change long in String in class Teacher ?
What do you think  ?

Reply:
Do you want to use string as datatype? I think it's more easier as we already worked with strings an we know most of the string's specific methods! Also I think long it's usually used for big math operations so I don't think it will be proper to use it for cnp. Let me know what you think!